api/swap/approval/_service.ts

```diff
@@ -68,6 +69,8 @@ export async function handleApprovalSwap(
       ? handleSwapBody(request.body, Number(request.query.destinationChainId))
       : { actions: [] };
 
+  const slippageTolerance = _slippageTolerance ?? slippage * 100;
```

Comment:
I think the null coalescing operator takes precedence right? Then we'd need the brackets so that `_slippageTolerance` doesn't get multiplied by 100 too.

```suggestion
  const slippageTolerance = _slippageTolerance ?? (slippage * 100);
```

Reply:
ohh good catch!